<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">setting
{
	background-color:#0a0612;
}
QPushButton
{
	background-color:#8a02b2;
	color:#1f1137;
	border-radius:20px;
}
QPushButton:hover
{
	background-color:#341d5c;
	color: #ffffff;
}
QLabel
{
	color:#ad03de;
}
QLineEdit
{
	background-color:#673ab7;
	color:#ffffff;
	border-radius:16px;
}
QLineEdit:hover
{
	background-color:#5d34a5;
}
QPlainTextEdit
{
	color:#ffffff;
	background-color:#3e236e;
	border-radius:12px;
}</string>
  </property>
  <widget class="QLabel" name="label_profile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>750</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="6" column="0">
     <widget class="QLabel" name="label_birthday">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Birth Day</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="pte_lastname">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="pte_biography">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="pte_name">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_biography">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Biography</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_email">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_name">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_year">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>    Year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pte_year">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_month">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Month</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pte_month">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_day">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>    Day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pte_day">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QPlainTextEdit" name="pte_email">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_gender">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_phonenumber">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>phone NO</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_lastname">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="pte_phonenumber">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_username">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="pte_username">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_edit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>720</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>70</y>
     <width>221</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_exit">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_logout">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_deleteaccount">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_messages">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Messages</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_changepassword">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_publicprivate">
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Public</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
